python/remotivelabs-broker/remotivelabs/broker/sync/client.py

```diff
@@ -124,6 +124,13 @@ def __init__(self, name: str, namespace: str):
         self.name = name
         self.namespace = namespace
 
+    @staticmethod
+    def parse(signal_id: str) -> SignalIdentifier:
+        s = signal_id.split(":")
+        if len(s) != 2:
+            raise BrokerException("signal names must have format namespace:signal_name")
```

Comment:
Should we include the signal_id in the exception?

Nit: Is it really a BrokerException? More like ValueError/IllegalArgument.